dist/znk.infra.js

```diff
@@ -4205,12 +4213,6 @@
                             _setDoneBtnDisplayStatus(newIndex);
                         });
 
-                        scope.$on(ZnkExerciseEvents.QUESTION_ANSWERED, function () {
```

Comment:
remove it, because it was showing the done button on click on the question before the last one, and also not needed i think, cause it shows the done button anyway when navigating to the last question.